docs/development/building-good-containers.md

```diff
@@ -0,0 +1,119 @@
+# Building Good Containers
+
+There are a number of recommendations based on our experience building
+Node.js applications in containers for deployments:
+
```

Comment:
We could include a snippet about building containers with alternatives to Docker (buildah, kaniko, podman).
Reference to: https://itnext.io/docker-kaniko-buildah-209abdde5f94
https://developers.redhat.com/blog/2019/02/21/podman-and-buildah-for-docker-users#what_is_buildah_and_why_would_i_use_it_-h2